mycroft/audio/speech.py

```diff
@@ -162,11 +127,15 @@ def mimic_fallback_tts(utterance, ident, listen):
         listen (bool): True if interaction should end with mycroft listening
     """
     if MimicTTSPlugin is not None:
-        tts = _get_mimic_fallback()
-        LOG.debug("Mimic fallback, utterance : " + str(utterance))
-        tts.execute(utterance, ident, listen)
-    else:
-        LOG.error("TTS FAILURE! utterance : " + str(utterance))
+        try:
+            tts = _get_mimic_fallback()
+            LOG.debug("Mimic fallback, utterance : " + str(utterance))
+            tts.execute(utterance, ident, listen)
+            return
+        except:
```

Comment:
Can we add a TODO here? Blanket catches mean somebody editing the called function is unable to crash loudly, but might not realize.

Reply:
will soon be deprecated in a follow up PR to make fallback configurable. it prints a log below about the error also